src/extends/PolicySelect/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Select } from '@b-design/ui';
+import { getPolicyList } from '../../api/application';
+import { PoliciesBase } from '../../interface/application';
+import locale from '../../utils/locale';
+import i18n from '../../i18n';
+
+type Props = {
+  onChange: (value: any) => void;
+  policySelectDataSource?: string[];
+  value: any;
+  id: string;
+  disabled: boolean;
+  appName: string;
+};
+
+type State = {
+  policySelectDataSource?: PoliciesBase[];
+};
+
+class PolicySelect extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      policySelectDataSource: [],
+    };
+  }
+
+  componentDidMount = () => {
+    this.fetchPolicyList();
+  };
+
+  fetchPolicyList = async () => {
+    const { appName = '' } = this.props;
+    getPolicyList({
+      name: appName,
+    })
+      .then((res) => {
+        if (res && res.policies) {
+          const policyListData = (res.policies || []).map((item: PoliciesBase) => item.name);
```

Comment:
We should show the policy with `name(type)` format

Reply:
done